api/src/main/java/org/apache/any23/vocab/Vocabulary.java

```diff
@@ -142,12 +142,12 @@ public IRI getProperty(String name, IRI defaultValue) {
      */
     public IRI getPropertyCamelCase(String property) {
         String[] names = property.split("\\W");
-        String camelCase = names[0];
+        StringBuilder camelCase = new StringBuilder(property.length()).append(names[0]);
         for (int i = 1; i < names.length; i++) {
             String tmp = names[i];
-            camelCase += tmp.replaceFirst("(.)", tmp.substring(0, 1).toUpperCase(java.util.Locale.ROOT));
+            camelCase.append(tmp.replaceFirst("(.)", tmp.substring(0, 1).toUpperCase(java.util.Locale.ROOT)));
```

Comment:
This would require that we import commons-lang3... is this something you are satisfied with considering that we were only just discussing dependency bloat. wdyt?

Reply:
We already have a dependency on commons-lang3, and it is being used all over our codebase. So I'm not too worried about that dependency in particular.

Reply:
I suppose it is true that this class in particular is part of any23 `api` rather than `core`, and `core` has the commons-lang3 dependency, so this would add that extra dependency to `api`. I'm not sure if this is a big problem since most people are using `core` rather than `api` alone... however if you'd rather not add an extra dependency to `api` I suppose we could leave it as-is for now (however I'd like to improve the implementation of that method at some point in the future).

Reply:
And now that I think about it more, I agree that there is definitely something to be said for keeping the dependencies of the `api` module as few in number as possible, so I think we should leave it as-is for now. Good call.

I'll mark this comment thread as 'resolved'.